### ISSUE TITLE
Export DATA_FILE path

### ISSUE THREAD

**Issue by jesseflorig** [freeall/mtgjson#2]
While using Meteor 1.3, I was having a hell of a time finding the path to the json file after the build process. In fact, I was only able to get to it via the absolute path. However when I changed the default export to `export.fetchJson` and added `export.jsonPath = DATA_FILE`, I'm able to easily find it for further parsing elsewhere.

I can submit a PR if you'd like to take a look.

Cheers.

**Comment by freeall** [comment]
Why do you want to find the json file and not just call the method and get the data back?

**Comment by jesseflorig** [comment]
When you call the method, doesn't it always attempt to re-fetch it? I suppose if you only wanted to read the cached file it would be helpful. But maybe not.

My initial problem was attempting to insert the data into Mongodb inside the callback which was giving me errors. I may just be better of refactoring my code to be fully asynchronous.

**Comment by freeall** [comment]
It asks the server for a new version, but if one exists, by using [etags](https://en.wikipedia.org/wiki/HTTP_ETag).

The idea with this module is that it will always return the newest data. In some cases that would be the local version, but then it would return that.

**Comment by freeall** [comment]
You should be able to insert the data into mongo without any issue. Maybe it's something else.

Out of mtg interest, what are you building? :)

**Comment by jesseflorig** [comment]
Thanks for your responses, I finally got it running correctly. I wasn't handling the callback correctly so my Mongo commands were failing when trying to run synchronously.

I'm working on an open source deck-builder/play-tester using Meteor/React. I have a few legacy toy projects up at Wzrdry.com that I abandoned to work on MtgDb.info. Unfortunately the project owner shut that down, and I'm just now restarting to build something after coming across Mtgjson.

I have a few prototypes that will be working their way into the final project:

http://codepen.io/jesseflorig/pen/rePaea
http://codepen.io/jesseflorig/pen/bpQxpJ
http://codepen.io/jesseflorig/pen/EKGxbx
